### ISSUE TITLE
kprocesshacker cannot be loaded on Insider Builds

### ISSUE THREAD

**Issue by sylveon** [processhacker/processhacker#1046]
**Describe the bug**
When booting up a recent Insider Build of Windows 11, a warning message appears, saying the driver cannot load because it was detected as a vulnerable driver.

On top, when opening Process Hacker, the `+` is missing from the window's title, meaning the driver is not available.

**To Reproduce**
1. Join the Insider Dev Ring
2. Upgrade to build 22509.1011
3. Turn on Core Integrity from the Windows Defender settings.
4. Reboot
5. See error

**Expected behavior**
The driver should not be blocked by Windows.

**Screenshots**
![image](https://user-images.githubusercontent.com/6440374/144686860-88e2bbb0-e24b-4d79-8678-6d2389964e95.png)
![image](https://user-images.githubusercontent.com/6440374/144686950-38cbb61d-3b75-4590-aad7-616c4b129f25.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 22509.1011

**Additional context**
This might require communication with Microsoft to figure what lead to it being blocked, resolve the issue, and then getting it unblocked from the OS.

**Comment by Biswa96** [comment]
Open some previous closed issues. You'll get the idea.

**Comment by sylveon** [comment]
This is different. While the previous issue was an issue with their computer, this is the OS explicitly blocking the kernel mode driver.

**Comment by sylveon** [comment]
Seems like a duplicate of #997 (I tried looking through the issue tracker before filling an issue, however my search didn't bring up this issue), but I ran Windows 10 and 11 release builds with core isolation active and the driver would not get blocked, so it's weird it only got blocked upon upgrading to Insider builds...
